app/controllers/v1/collections_controller.rb

```diff
@@ -15,6 +15,7 @@ def index
     #
     def show
       @collection = Purl.find_by_druid(druid_param)
+      raise ActiveRecord::RecordNotFound if @collection.blank?
```

Comment:
I think you can do this by just adding `!` to the end of `Purl.find_by_druid` 

See http://api.rubyonrails.org/classes/ActiveRecord/FinderMethods.html#method-i-find_by-21